src/_resources/slide-deck-ca-transit-data-guidelines-draft-v3.0.md

```diff
@@ -1,7 +1,7 @@
 ---
 date: "2022-10-01T17:00:00-07:00"
 title: "Overview: How Cal-ITP incorporated public feedback into CA Transit Data Guidelines v3.0"
-asset: "Cal-ITP.Transit.Draft.Guidelines.Draft.3.Feedback.pdf"
+asset: "Cal-ITP.Transit.Draft.Guidelines.Feedback.pdf"
```

Comment:
@mrose914 is this change okay? The updated file you gave me had the updated name without the extra `Draft.3` so I kept that name. Just to let you know, if this PDF has been linked to by another site or page, this name change could break those links.

Reply:
I checked on Google Search Console and other backlinks checkers to see if there are any websites that link to https://www.calitp.org/assets/Cal-ITP.Transit.Draft.Guidelines.Draft.3.Feedback.pdf. Then, I checked in the Cal-ITP Slack to see if this URL is linked or referenced there -- no references. So I feel comfortable changing this link. 

In an ideal future scenarios, we should reject URL filenames with Draft in them.

Reply:
Wait @machikoyasuda @allejo I think this was incorrect, or at least wasn't finished all the way. 

This links to a deck on Cal-ITP's work on the Draft v3 Transit Data Guidelines (e.g. the working version of v3 when it was in-progress but v2 was still current). This deck is highlighting feedback Cal-ITP took during the development of this v3 update.

In any case, `Draft` is still in the file name, but now we lost the `3` version number.

Reply:
Ooooh... oops. It looks like we are waiting on GDrive access to this PDF still?

Reply:
Wait. No. I see we do have the PDF here now: https://github.com/cal-itp/calitp.org/blob/6c97915c3d49326746d0bf04e2d77a60c967e0ca/src/assets/Cal-ITP.Transit.Draft.Guidelines.Feedback.pdf

I see the PDF itself has the title: CA Transit Data Guidlines Draft v3.0 Feedback, so it seems to me like that would be the best URL slug?

Reply:
Will make a new issue and PR to fix